web/forms.py

```
@@ -1787,3 +1792,101 @@ class StudyGroupForm(forms.ModelForm):
+
+    def clean_features(self) -> list:
+        features = self.cleaned_data.get("features")
+        if features:
+            try:
+                if isinstance(features, str):
+                    import json
+
+                    features = json.loads(features)
+
+                if not isinstance(features, list):
+                    raise forms.ValidationError("Features must be a list of items")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)